docs/source/runtime_troubles.rst

```diff
@@ -8,10 +8,10 @@ version of Python considers legal code. This section describes these scenarios
 and explains how to get your code running again. Generally speaking, we have
 three tools at our disposal:
 
-* Use of ``from __future__ import annotations`` (:pep:`563`)
-  (this behaviour may eventually be made the default in a future Python version)
 * Use of string literal types or type comments
 * Use of ``typing.TYPE_CHECKING``
+* Use of ``from __future__ import annotations`` (:pep:`563`)
+  (This directive has been declared deprecated since Python 3.14, and is expected to be removed in a future version of Python)
```

Comment:
Maybe mention PEP 649 or PEP 749 explicitly here? There were quite a few discussions regarding deferred annotations, I suspect some documentation users may struggle to find the final outcome:)

But also, given that 3.14 is a pre-release, maybe it's better to wait until we actually have 3.14.0 released. While extremely unlikely, things may still change, and this deprecation may get deferred again.